src/IdServer/SimpleIdServer.IdServer/ServiceCollectionExtensions.cs

```diff
@@ -345,6 +376,12 @@ private static IServiceCollection AddRegisterApi(this IServiceCollection service
             return services;
         }
 
+        private static IServiceCollection AddSeeders(this IServiceCollection services)
```

Comment:
Since version 5.0.0, I removed the dependency between the "SimpleIdServer.IdServer" library and "SimpleIdServer.IdServer.Store." 
Now there are two libraries: "SimpleIdServer.IdServer.Store.EF" and "SimpleIdServer.IdServer.Store.SqlSugar." 
Each package supports a different ORM (EF and SqlSugar). Therefore, the code for "AddSeeders" cannot be located in "SimpleIdServer.IdServer" but must be in both packages "SimpleIdServer.IdServer.Store.EF" and "SimpleIdServer.IdServer.Store.SqlSugar.

Reply:
I take the `master` branch as base code, but the new projects appeared soon. However, I don’t have any issues with this.

Reply:
I created DTOs, I should do the same with these?